bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/DaikinBindingConstants.java

```diff
@@ -42,5 +46,17 @@
     public static final String CHANNEL_AC_FAN_DIR = "fandir";
     public static final String CHANNEL_HUMIDITY = "humidity";
 
-    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_AC_UNIT);
+    // additional channels for Airbase Controller
+    public static final String CHANNEL_AIRBASE_AC_FAN_SPEED = "airbasefanspeed";
+    public static final String CHANNEL_AIRBASE_AC_ZONE1 = "zone1";
```

Comment:
You should be able to remove all these individual zone channels with a generic: `CHANNEL_AIRBASE_AC_ZONE = "zone";` . See my other comments.

Reply:
Fixed in 4cafb310da

Reply:
Can you remove the constants in comment. There not needed any more I assume.